### PR TITLE
Know its peer DOTS agents

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

File: draft-ietf-dots-requirements.md [hunk 1]
```diff
@@ -243,7 +243,7 @@ communication channel between DOTS agents. Indeed, establishing a relationship
 with peer DOTS agents during normal network conditions provides the foundation
 for more rapid attack response against future attacks, as all interactions
 setting up DOTS, including any business or service level agreements, are already
-complete.
+complete. Peer DOTS agents are provisioned to a DOTS client using a variety of means (e.g., manual or dynamic).
```
Comment (amortensen) [inline draft-ietf-dots-requirements.md]: Can we reword to something like: `Peer DOTS agents are provisioned to a DOTS client using a variety of manual or dynamic methods`?

File: draft-ietf-dots-requirements.md [hunk 2]
```diff
@@ -277,14 +277,14 @@ another vector for the very attacks it's meant to help fight off. DOTS clients
 must be able to authenticate DOTS servers, and vice versa, to avoid exposing new
 attack surfaces when deploying DOTS; specifically, to prevent DDoS mitigation in
 response to DOTS signaling from becoming a new form of attack. In order to
-provide this level of proteection, DOTS agents must have a way to negotiate and
+provide this level of protection, DOTS agents must have a way to negotiate and
```
Comment (amortensen) [inline draft-ietf-dots-requirements.md:280]: I just merged this change in from Flemming's pull request (PR #65), you can drop it from the next revision of this PR.

**Review by amortensen — CHANGES_REQUESTED** [review]
Looks fine, just a couple minor things.

File: draft-ietf-dots-requirements.md (hunk 2 above)
Comment (boucadair) [reply draft-ietf-dots-requirements.md:280]: Works for me

**Review by amortensen — APPROVED** [review]

**Pull request merged**